File: app/src/main/java/com/example/myappcontacts/business/contactsmap/IContactsMapInteractor.java
package com.example.myappcontacts.business.contactsmap;

import com.example.myappcontacts.data.dao.contactsmap.MapMarkersModel;

import java.util.List;

import io.reactivex.Single;

public interface IContactsMapInteractor {

    Single<List<MapMarkersModel>> loadMapMarkersList();
}

File: app/src/main/java/com/example/myappcontacts/business/contactsmap/IContactsMapInteractor.kt
package com.example.myappcontacts.business.contactsmap

import com.example.myappcontacts.data.dao.contactsmap.db.MapMarkersModel

import io.reactivex.Single

interface IContactsMapInteractor {

    fun loadMapMarkersList(): Single<List<MapMarkersModel>>
}
